### ISSUE TITLE
Holes and Defects on the side of the Print

### ISSUE THREAD

**Issue by daniel77261** [Sielu-Rooster/DeckGuard#1]
Hello Sielu-Rooster,

I have a AnyCubic I3 Mega and for the most part it prints pretty flawlessly.

I am currently trying to print this SteamDeck Guard and having issues with holes and defects on the side of the print.

It always happens at the same place (to the left of the indented hexagon) and it gets to a certain level and then it's completely fine and goes back to being flawless. Here is what I've tried so far:

• Levelled the print bed.
• Cleaned the print bed.
• Changed the filament type (eSUN PLA+ , Kexcelled PLASilk).
• Printed on the floor for stability.
• Changed the print temperature (200 - 215 degrees).
• Changed the print speed (50% - 100%).
• Changed the infill denstity (25% - 80%).

I have never experienced this issue before and everything else seems completely fine. My next steps are to change the retraction settings and reorient the print to see if this works.

Any assistance or troubleshooting will be really appreciated. 
![Reference](https://user-images.githubusercontent.com/81775572/180208529-4f5aaf83-694f-4a39-81b4-affa1224adab.png)

**Comment by Sielu-Rooster** [comment]
It's been a while since I've worked with FDM printers, and I have seen this before on other prints but I don't remember how to fix it.... so take any advice here with a big ol' grain of salt.  

Is the print wobbling or print head wobbling when the head passes over this area?  Did speeding up the print make it worse?  I suspect something is happening when the head goes around the sharp angles of the indent, either in the print itself or in the head.  Slowing down _should_ improve it, but I'm not sure.  Are you able to print the one-piece version with supports?  Do you see the same thing on that STL in the same area?

**Comment by Sielu-Rooster** [comment]
Before you start a new print, can you post a section of the G-Code outline during this segment?  I'd like to see what path the print head is taking over this feature.

**Comment by daniel77261** [comment]
Hey Sielu-Rooster,

Thanks for your prompt reply and help.

I've not seen any such wobble but I'll do another test and pay close attention.

I believe the speed being slower helps the print, I'll give 20mm/s a try and see how this looks.

I'll send you a copy of the G-Code shortly.

Thank you!

**Comment by daniel77261** [comment]
Hey Sielu-Rooster,

I hope this is what you need with regards to the G-code, please see attached.
[G-code.txt](https://github.com/Sielu-Rooster/DeckGuard/files/9172678/G-code.txt)

**Comment by Sielu-Rooster** [comment]
That looks like a log from the printer... I think I can extract the code from it but it's tedious, can you send me the raw g-code file from the slicer?  Did slowing down the print do anything for you?

**Comment by daniel77261** [comment]
Sorry about that, see the text file attached.

[DeckGuard_Left.txt](https://github.com/Sielu-Rooster/DeckGuard/files/9173883/DeckGuard_Left.txt)

Slowing it it down definitely helped. I set the print speed to 20mm/s and printed a test, see the side by side comparison attached.

![IMG_3583](https://user-images.githubusercontent.com/81775572/180607543-80266186-5c48-4cad-bd9b-c99f433e8ef3.jpg)

**Comment by daniel77261** [comment]
I've spent a lot of time investigating this now, changing a lot of basic settings and I believe this an issue to do with the infill settings specifically with Ultimaker Cura slicer.

I successfully printed the DeckGuard_Left piece using Kexcelled Blue PLASilk with the following profile settings:

**Profile Reference:** Normal - 0.2mm
**Support:** Yes
**Minimum Extrusion Distance Window:** 0.0mm
**Printing Temperature:** 200° C
**Print Speed:** 60.0mm/s
**Infill Density:** 0%

![Successful Cura Print](https://user-images.githubusercontent.com/81775572/180626479-6699cc03-6846-47c8-b874-450b0436b268.jpg)

Every print kept having the same artifacts regardless of what settings I changed until I dropped the infill density to 0%.

I believe what is happening is that the walls are so thin that when the printer attempts to infill, it moves so quickly between each side of the wall that it overshoots and adds artifacts to the outside of the wall. Coupled with the indented hexagon shape who's complexity only exacerbates the issue.

I had a friend attempt the same print with a Original Prusa i3 printer using PrusaSlicer and this printed without any artifacts. He then copied my Ultimaker Cura slicer profile and sliced the model with Ultimaker Cura slicer and was able to replicate the issue on a Original Prusa i3.

![Comparison](https://user-images.githubusercontent.com/81775572/180626535-4d83ee12-3db8-40ad-9a85-e271f715c4e1.png)

This is a comparison of two pieces he printed on a Original Prusa i3. One was sliced in PrusaSlicer which printed perfectly and one was sliced in Ultimaker Cura slicer which printed with artifacts.

I am currently printing another test on my Anycubic I3 Mega printer using PrusaSlicer to see if this prints as desired. The other test would be to change the infill pattern to something with less edges.
